### DISCUSSION TITLE
error message duplicated in source code

### DISCUSSION THREAD

**Discussion by laoshaw** [w0rp/ale#4419] [Q&A - Ask for help with problems] [answered]
I had warnings and errors in my quickfix window but suddenly they're shown in the source code directly, I don't need they are displayed at two places with the same info, how can I disable the error messages in the source code window? it causes the code hard to read too.

**Comment by laoshaw** [comment, marked as answer]
ok I need do this: let g:ale_virtualtext_cursor=0 to get rid of that

**Reply by gcaprio** [reply]
thank you! been searching for this forever.
